core/modules/layout/css/layout.admin.css

```diff
@@ -97,11 +97,15 @@
 
 /* Buttons that are displayed as links. */
 .layout-access-remove,
-.layout-access-configure {
+.layout-access-configure,
```

Comment:
We could add some common class (like `.layout-action-link`) to all these links instead of having this list of classes grow.

Reply:
This would affect existing sites where the class is already in use.